image_vise.gemspec

```
@@ -22,25 +22,27 @@ Gem::Specification.new do |spec|
     raise "RubyGems 2.0 or newer is required to protect against public gem pushes."
   end
 
+  spec.required_ruby_version = ">= 2.7"
+
   spec.files         = `git ls-files -z`.split("\x0")
   spec.bindir        = "exe"
   spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
   spec.require_paths = ["lib"]
 
   spec.add_dependency 'patron', '~> 0.9'
   spec.add_dependency 'rmagick', '~> 3'
-  spec.add_dependency 'ks'
   spec.add_dependency 'rack', '>= 1', '< 3'
   spec.add_dependency 'format_parser', '~> 0.25'
   spec.add_dependency 'measurometer', '~> 1'
+  spec.add_dependency "addressable"
```

```suggestion
  spec.add_dependency 'addressable', '>= 2', '< 3'
```

What about setting a range of versions that we trust that it will work? (the current version is 2.8)